main.go

```diff
@@ -604,3 +617,10 @@ func generatePackageDetailsHTMLElements(testSuiteOverview map[string][]string, p
 
 	return strings.Join(elem, "\n"), nil
 }
+
+func formatTimeDisplay(secs float64) (float64, string) {
+	if secs > 1 {
+		return secs, "s"
+	}
+	return secs * float64(1000), "ms"
```

Comment:
Think we can potentially use the time.ParseDuration function. To convert from seconds to milliseconds. This could potentially remove the long decimal places, of which mostly are just zeros